docs/reference/indices/split-index.asciidoc

```diff
@@ -0,0 +1,165 @@
+    for a single split operation.
+
+In order to split an index, the index must be marked as read-only,
+and have <<cluster-health,health>> `green`.
+
+This can be achieved with the following request:
+
+[source,js]
+--------------------------------------------------
+PUT /my_source_index/_settings
+{
+  "settings": {
+    "index.blocks.write": true <1>
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+<1> Prevents write operations to this index while still allowing metadata
+    changes like deleting the index.
+
+[float]
+=== Spitting an index
+
+To split `my_source_index` into a new index called `my_target_index`, issue
+the following request:
+
+[source,js]
+--------------------------------------------------
+POST my_source_index/_split/my_target_index
+{
+  "settings": {
+    "index.number_of_shards": 2
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+The above request returns immediately once the target index has been added to
+the cluster state -- it doesn't wait for the split operation to start.
+
+[IMPORTANT]
+=====================================
+
+Indices can only be split if they satisfy the following requirements:
+
+* the target index must not exist
+
+* The index must have less primary shards than the target index.
+
+* The number of primary shards in the target index must be a power of two
+  factor of the number of primary shards in the source index.
+
+* The node handling the split process must have sufficient free disk space to
+  accommodate a second copy of the existing index.
+
+=====================================
+
+The `_split` API is similar to the <<indices-create-index, `create index` API>>
+and accepts `settings` and `aliases` parameters for the target index:
+
+[source,js]
+--------------------------------------------------
+POST my_source_index/_split/my_target_index
+{
+  "settings": {
+    "index.number_of_shards": 5 <1>
+  },
+  "aliases": {
+    "my_search_indices": {}
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[s/^/PUT my_source_index\n{"settings": {"index.blocks.write": true, "index.number_of_routing_shards" : 5, "index.number_of_shards": "1"}}\n/]
+
+<1> The number of shards in the target index. This must be a factor of the
+    number of shards in the source index.
+
+
+NOTE: Mappings may not be specified in the `_split` request, and all
+`index.analysis.*` and `index.similarity.*` settings will be overwritten with
+the settings from the source index.
+
+[float]
+=== Monitoring the split process
+
+The split process can be monitored with the <<cat-recovery,`_cat recovery`
+API>>, or the <<cluster-health, `cluster health` API>> can be used to wait
+until all primary shards have been allocated by setting the  `wait_for_status`
+parameter to `yellow`.
+
+The `_split` API returns as soon as the target index has been added to the
```

Comment:
why don't we wait for primaries to be allocated like we do during normal index creation?

Reply:
I don't know that is something for later... it's the same as for the _shrink API

Reply:
as far as I can tell MetaDataCreateIndexService#createIndex and the rest parsing we do the default, which means waiting for the primaries to be active...

Reply:
ok then it's legacy and needs to be removed